tests/Benchmarks/Utf8String/CodePointEnumeration.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Code;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Utf8StringRealType = System.Text.Utf8.Utf8String;
+
+namespace Benchmarks.Utf8String
```

Comment:
`Benchmarks.SystemTextUtf8String` (with the folder name changed to `Benchmarks/SystemTextUtf8String/`), similar to https://github.com/dotnet/corefxlab/pull/2249/files#diff-9803e8894ed7238070c254047cf81cf7R13?

Reply:
I am fine with that.